invenio_app_ils/circulation/utils.py

```diff
@@ -43,6 +43,14 @@ def circulation_document_retriever(item_pid):
     return item["document"]["document_pid"]
 
 
+def circulation_item_retriever(item_pid):
```

Comment:
I think this already done in a previous commit https://github.com/inveniosoftware/invenio-app-ils/commit/9a23864f7e2ef573440cd66c05a508e1811aa32e. Can you delete that?